godot-core/src/builtin/quaternion.rs

```diff
@@ -281,6 +281,15 @@ impl Mul<Quaternion> for Quaternion {
     }
 }
 
+impl Mul<Vector3> for Quaternion {
+    type Output = Vector3;
+
+    fn mul(self, rhs: Vector3) -> Self::Output {
+        // TODO: should we panic if this quaternion is not normalized
```

Comment:
What happens if we _don't_ panic?

Reply:
oh I probably should be more clear in this in that glam will panic if the quaternion is not normalized. It was more of a should we attempt to normalize the quaternion ourselves before passing to glam

Reply:
No, I think it's fine.

But if this operation can panic, it would be good that instead of a `TODO` comment, you'd have a `/// # Panics` documentation that describes the condition under which a panic can occur.

Reply:
Alright i updated the documentation here then